llvm/lib/Transforms/Coroutines/CoroSplit.cpp

```diff
@@ -1455,6 +1462,62 @@ struct SwitchCoroutineSplitter {
     setCoroInfo(F, Shape, Clones);
   }
 
+  static Function *createNoAllocVariant(Function &F, coro::Shape &Shape,
+                                        SmallVectorImpl<Function *> &Clones) {
+    auto *OrigFnTy = F.getFunctionType();
```

Comment:
If I am not mistaken, the identation looks not good here.

Reply:
Didn't get that?

Reply:
Never mind. I made a silly mistake.